src/Command/PqCommands/ContentHubPqDependencies.php

```diff
@@ -0,0 +1,378 @@
+    $this->addOption('entity-type', 'et', InputOption::VALUE_OPTIONAL, 'Run checks only for the provided entity type. Specify bundle if needed. Example: {entity_type}:{bundle},{entity_type2}.', '');
+    $this->addUsage('ach:pq:dependencies --entity-type "node:article,node:page,block_content"');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function runCommand(InputInterface $input, PqCommandResult $result): int {
+    if (!$this->depcalcModuleIsEnabled()) {
+      throw ContentHubPqCommandErrors::newException(ContentHubPqCommandErrors::$depcalcIsNotEnabledError);
+    }
+
+    $this->entityTypesFilter = $this->parseInput($input);
+
+    $contentEntities = $this->getExportEligibleContentEntities();
+    $contentEntityCount = $this->countEligibleEntities($contentEntities);
+    $formatted = [];
+    foreach ($contentEntityCount as $key => $val) {
+      $formatted[] = sprintf('%s: %s ', $key, $val);
+    }
+    $result->setIndicator(
+      'Export Eligible Content Entities',
+      implode("\n", $formatted),
+      'The following entities and their count should be expected to be exported. The list does not contain dependencies. Make sure to only export desired entities.'
+    );
+
+    $depCount = $this->calculateDependenciesForAllEligibleContentEntities($contentEntities);
+    $messages = [
+      'positive' => 'The number of eligible dependencies. The overall entity count that would be exported, content and config entities included.',
+      'negative' => PqCommandResultViolations::$dependencyCount,
+    ];
+    $high = $depCount >= 5000;
+    $result->setIndicator(
+      'Dependencies',
+      $depCount,
+      $high ? $messages['negative'] : $messages['positive'],
+      $high
+    );
+
+    $rowsShouldBeIncreased = $depCount >= $this->getMaxRowsForDepcalcCache();
+    $messages = [
+      'positive' => 'The configured maximum number of depcalc cache bin rows is sufficient.',
+      'negative' => PqCommandResultViolations::$depcalcCacheMaxRows,
+    ];
+    $result->setIndicator(
+      'Depcalc Cache Bin Max Rows',
+      $this->getMaxRowsForDepcalcCache(),
+      $rowsShouldBeIncreased ? $messages['negative'] : $messages['positive'],
+      $rowsShouldBeIncreased
+    );
+
+    return 0;
+  }
+
+  /**
+   * Returns an array of all export eligible content entities.
+   *
+   * @return array
+   *   A list of eligible entities.
+   *
+   * @throws \Exception
+   */
+  public function getExportEligibleContentEntities(): array {
+    $entityTypeManager = $this->drupalServiceFactory->getDrupalService('entity_type.manager');
+    $entities = [];
+    foreach ($entityTypeManager->getDefinitions() as $entityType) {
+      if (!$this->isEligible($entityType)) {
+        continue;
+      }
+      $entities[] = $entityType;
+    }
+
+    return $entities;
+  }
+
+  /**
+   * Counts eligible content entities.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityTypeInterface[] $entities
+   *   The eligible entities to count.
+   *
```

Comment:
Please add `@return` doc hint.

Reply:
hmm, weird, I thought my git hook would catch that.